client/dashboard/sites/logs/utils.ts

```diff
@@ -2,7 +2,21 @@ import { dateI18n } from '@wordpress/date';
 import { __, sprintf } from '@wordpress/i18n';
 import { startOfDay, endOfDay, fromUnixTime, isValid as isValidDate } from 'date-fns';
 import { formatDateWithOffset, getUtcOffsetDisplay } from '../../utils/datetime';
-import type { PHPLog } from '@automattic/api-core';
+import type { PHPLog, ServerLog } from '@automattic/api-core';
+
+/**
+ * Simple throttle function to limit how often a function can be called.
+ */
+export function throttle( fn: () => void, limit: number ) {
```

Comment:
we can import it from `@wordpress/compose`

Reply:
Oh! Thanks for spotting that.